### PR TITLE
add .gitattributes

### PR DESCRIPTION
**Pull request by kamilsk**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+build/*          export-ignore
+docs/*           export-ignore
+tests/*          export-ignore
+.editorconfig    export-ignore
+.gitattributes   export-ignore
+.gitignore       export-ignore
+.travis          export-ignore
```
Comment (Aliance) [inline .gitattributes]: There is no `.travis` file, only `.travis.yml` – is this a bug or just `.travis` consumed as file prefix, so `.travis.yml` suits this pattern?

Comment (kamilsk) [inline .gitattributes]: this is a mistake, you are right @Aliance

Comment (kamilsk) [inline .gitattributes]: Yes, you can fix it. Thank you!

Comment (Aliance) [inline .gitattributes]: #1649

**Pull request merged**